common/offmap_powers/policy.cwt

```diff
@@ -0,0 +1,19 @@
+# TODO: Docs
+# ## Policies are modifiers so they can contain the same things are other modifiers
+# however they are not applied to a character but to the offmap_power
```

Comment:
This and statuses already exist in `common/offmap.cwt` proper. No need to duplicate here.